File: 해시/헌석/완주하지_못한_선수.py
def solution(participant, completion):
    
    people = dict()
    
    for i in participant:
        if i in people:
            people[i]+=1
        else:
            people[i] = 1
    
    for i in completion:
        if people[i] > 1:
            people[i]-=1
        else:
            people.pop(i)
    
    for i in people:
        answer = i
    return answer

File: 해시/김지후/완주하지_못한_선수.js
function solution(participant, completion) {
    const sortParticipant = participant.sort();
    const sortCompletion = completion.sort();

    for (let i = 0; i <= sortParticipant.length; i++) {
        if (sortParticipant[i] !== sortCompletion[i]) {
            return sortParticipant[i]
        }
    }
}